src/constants/account.ts

```diff
@@ -45,3 +48,12 @@ export type EvmDerivedAddresses = {
 };
 
 export const AMOUNT_RESERVED_FOR_GAS_USDC = 0.1;
+
+export type LocalOrderData = {
+  marketId: string;
+  clientId: number;
+  orderId?: string;
+  orderType?: TradeTypes;
+  submissionStatus: OrderSubmissionStatuses;
```

Comment:
just double checking my own understanding, is `orderId` null when the order fails? When is `orderType` null?

Reply:
ah yes orderType shouldn't be null!
orderId is only created if the order's been indexed, so it won't exist when order was just placed or errored!